### PR TITLE
issue #578 Error message UI fix

### LINKED ISSUE #578: Error message UI fix.
**Describe the bug**
Text is not aligned correctly when Contributors are not loaded properly and looks something like this. 

Screenshot 2020-03-04 at 9 34 30 PM

**Expected behavior**
It should be aligned properly with proper font size. Try it in your way, or let me know if you need any help with the design.

### PR DESCRIPTION
**Pull request by shreyakapoor08**
Fixes #578 

####Description
Error Message UI corrections

ignus

### CODE REVIEW

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 change the font color of `Something went wrong!` to `#000066`, add `20px` margin at the bottom of button and change the font-size of details tab to `15px`.
> 
> After pressing the button, it shows the extended border. That should just be around the button, just a bit of padding would be fine.

Corrections made, please check

File: src/ignitus-Internals/index.js [hunk 1]
```diff
@@ -24,8 +24,9 @@ export class ErrorBoundary extends React.Component {
     if (errorInfo) {
       return (
         <div className="container">
-          <h2>Something went wrong.</h2>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
+          <h2 style={{ textAlign: 'center', marginTop: '40px', color: '#000066' }}>Unable to load the contributors!</h2>
+          <h5 style={{ textAlign: 'center', marginTop: '20px' }}>This can happen if you are not connected to internet or if and underlying<br/>system or component is not available</h5>
+          <details style={{ whiteSpace: 'pre-wrap', textAlign: 'center', fontSize: '15px', marginBottom: '20px', paddingLeft: '380px', paddingRight: '380px' }}>
```
Comment (gittysachin) [inline src/ignitus-Internals/index.js]: Please change the padding on both the sides to `210px` to be on the safe side if the error text lines are larger. That way text will look nicer.

File: src/ignitus-Api/constants.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ export const TESTIMONIAL_URL = `${BASE_URL}/testimonials`;
 export const STUDENT_SIGN_UP = `${BASE_URL}/register/student`;
 export const STUDENT_SIGN_IN = `${BASE_URL}/login`;
 
-export const FRONTEND_CONTRIBUTOR_API = 'https://api.github.com/repos/Ignitus/Ignitus-Client-Side-Development/contributors?per_page=14';
+export const FRONTEND_CONTRIBUTOR_API = 'https://api.githubb.com/repos/Ignitus/Ignitus-Client-Side-Development/contributors?per_page=14';
```
Comment (divyanshu-rawat) [inline src/ignitus-Api/constants.ts]: Please revert it in the last commit once we approve the PR, its good for testing :) Thanks.

File: src/ignitus-Internals/index.js [hunk 3]
```diff
@@ -24,8 +24,16 @@ export class ErrorBoundary extends React.Component {
     if (errorInfo) {
       return (
         <div className="container">
-          <h2>Something went wrong.</h2>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
+          <h2
+           style={{ textAlign: 'center', marginTop: '3em', color: '#000066' }}>
+            Unable to load the contributors!
+          </h2>
+          <h5 style={{ textAlign: 'center', marginTop: '20px' }}>
```
Comment (divyanshu-rawat) [inline src/ignitus-Internals/index.js]: Please avoid inline-styling try to write css as a part of styles file.

**Comment by divyanshu-rawat** [comment]
@Paarmita feel free to review the design here - https://deploy-preview-598--peaceful-nightingale-55a1b7.netlify.com/aboutus.

**Comment by divyanshu-rawat** [comment]
@shreyakapoor08 can you please try placing `Details` to the left, as I think placing it in the centre has also centred the content inside it.

Screenshot 2020-03-11 at 15 18 23

File: src/ignitus-Internals/index.js (hunk 3 above)
Comment (shreyakapoor08) [reply src/ignitus-Internals/index.js]: > Please avoid inline-styling try to write css as a part of styles file.

okay I will take care of it

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 can you please try placing `Details` to the left, as I think placing it in the centre has also centred the content inside it.
> 
> Screenshot 2020-03-11 at 15 18 23

okay I will just do that

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 can you please try placing `Details` to the left, as I think placing it in the centre has also centred the content inside it.
> 
> Screenshot 2020-03-11 at 15 18 23

Please check now, I have aligned the details tab to the left

**Comment by divyanshu-rawat** [comment]
@shreyakapoor08 It would be great if you fix the api url in the end, because until then how can we approve it? that's what I mentioned here https://github.com/Ignitus/Ignitus-client/pull/598#discussion_r391001335

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 It would be great if you fix the api url in the end, because until then how can we approve it? that's what I mentioned here [#598 (comment)](https://github.com/Ignitus/Ignitus-client/pull/598#discussion_r391001335)

https://github.com/Ignitus/Ignitus-client/pull/598/commits/615d2735776fc8e1231238a3af565b7370064c1c
kindly check this "final correction" commit 
I have corrected the api url also

**Comment by shreyakapoor08** [comment]
> > @shreyakapoor08 It would be great if you fix the api url in the end, because until then how can we approve it? that's what I mentioned here [#598 (comment)](https://github.com/Ignitus/Ignitus-client/pull/598#discussion_r391001335)
> 
> [615d273](https://github.com/Ignitus/Ignitus-client/commit/615d2735776fc8e1231238a3af565b7370064c1c)
> kindly check this "final correction" commit
> I have corrected the api url also

https://github.com/Ignitus/Ignitus-client/pull/598/commits/a7f602fb3f1e9649aa32db72c3c0296f4138dae3

File: src/ignitus-Internals/index.js [hunk 4]
```diff
@@ -24,8 +24,16 @@ export class ErrorBoundary extends React.Component {
     if (errorInfo) {
       return (
         <div className="container">
-          <h2>Something went wrong.</h2>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
+          <h2
+           style={{ textAlign: 'center', marginTop: '4em', color: '#000066' }}>
```
Comment (divyanshu-rawat) [inline src/ignitus-Internals/index.js]: Inline styles are still there, please create a seperate `.scss file` and place them there and import it in this component.

File: src/ignitus-Internals/index.js [hunk 5]
```diff
@@ -24,8 +24,16 @@ export class ErrorBoundary extends React.Component {
...
+            Unable to load the contributors!
+          </h2>
+          <h5 style={{ textAlign: 'center', marginTop: '20px' }}>
```
Comment (divyanshu-rawat) [inline src/ignitus-Internals/index.js]: same for this, place it in separate .scss file.

File: src/ignitus-Internals/index.js [hunk 6]
```diff
@@ -24,8 +24,16 @@ export class ErrorBoundary extends React.Component {
...
+            This can happen if you are not connected to internet or if and underlying
+            <br />
+            system or component is not available
+          </h5>
+          <details style={{ whiteSpace: 'pre-wrap', fontSize: '15px', marginBottom: '4em', marginRight: '530px', marginLeft: '220px' }}>
```
Comment (divyanshu-rawat) [inline src/ignitus-Internals/index.js]: Same 🔝

**Comment by divyanshu-rawat** [comment]
@shreyakapoor08 please show us how the final design looks like, you can attach a screenshot here, then I can review 👍

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 please show us how the final design looks like, you can attach a screenshot here, then I can review 👍

here it is

errormessage

**Comment by shreyakapoor08** [comment]
> LGTM :). Thanks @Paarmita please have a look, I will merge it tonight to develop and resolve the merge conflicts myself, thanks for your contribution 👍

Thankyou :) 
I have a few doubt regarding keeping my branch updated with the master branch. will ask about it on slack

**Comment by divyanshu-rawat** [comment]
> > LGTM :). Thanks @Paarmita please have a look, I will merge it tonight to develop and resolve the merge conflicts myself, thanks for your contribution 👍
> 
> Thankyou :)
> I have a few doubt regarding keeping my branch updated with the master branch. will ask about it on slack

Sure you are welcome :)

**Comment by shreyakapoor08** [comment]
@divyanshu-rawat @Paarmita Difficulty tag is not added due to which i am not able to see my updated score. Kindly check

**Comment by divyanshu-rawat** [comment]
@shreyakapoor08 I have just added them thanks for reminding :)

**Comment by shreyakapoor08** [comment]
Thank you so much 😊

**Pull request merged**